packages/babel-preset/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "private": true,
+  "name": "@hig/babel-preset",
+  "version": "0.1.0",
+  "description": "HIG Babel preset",
+  "author": "Autodesk Inc.",
+  "license": "Apache-2.0",
+  "homepage": "https://hig.autodesk.com",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/Autodesk/hig"
+  },
+  "main": "index.js",
+  "peerDependencies": {
```

Comment:
Not a true dependency?

Reply:
You're right, we don't want to have to list each plugin and preset as a `devDependency` for each package.

However, if we want to be certain that it's using the correct plugin, we should import them directly [similar to how Airbnb does it](https://github.com/airbnb/babel-preset-airbnb/blob/master/index.js#L31). We don't need to do that right now though.